packages/scandipwa/src/util/Validator/Validator.js

```diff
@@ -196,4 +196,47 @@ export const validateGroup = (DOM, rule = null) => {
     return errorMessages.length === 0 && errorFields.length === 0 ? true : output;
 };
 
+/**
+ * Get number of different character classes
+ * @param value
```

Comment:
Maybe it worth to add types to parameters as well since we are adding return types?
Otherwise all arguments shows as `any` type but you expect strings or number in them without telling an editor about it :thinking:

yeah, dunno why it didn't caught my eye last time I reviewed your PR...